/**:
  ros__parameters:
    tolerance: 0.7
    voxel_leaf_size: 0.3
    min_points_number_per_voxel: 1
    min_cluster_size: 10
    max_cluster_size: 3000
    use_height: false
    input_frame: "base_link"

    # low height crop box filter param
    max_x: 200.0
    min_x: -200.0
    max_y: 200.0
    min_y: -200.0
    max_z: 2.0
    min_z: -10.0
    negative: false
